input/templates/partials/myaccount/my-address-book-adress.hbs

```diff
@@ -0,0 +1,18 @@
+<div>{{address.title}} {{address.firstName}} {{address.lastName}}</div>
+<div>{{address.streetName}}</div>
+<div>{{address.additionalStreetInfo}}</div>
+<div>{{address.region}}</div>
+<div>{{address.postalCode}}</div>
+<div>{{address.country}}</div>
+<div class="address-book-edit-delete">
+  <a href="{{address.editUrl}}">
+    <img src="{{meta.assetsPath}}/img/edit-1.png" alt="edit icon">
+    {{i18n "my-account:buttons.edit"}}
+  </a>
+</div>
+<div>
+  <a href="{{address.deleteUrl}}">
+    <img src="{{meta.assetsPath}}/img/delete-60px.png" alt="delete icon">
+    {{i18n "my-account:buttons.delete"}}
```

Comment:
I have forced that all addresses, including default shipping/billing addresses, can be removed. Was there any reason why they were not offered this option for the default addresses? I can imagine that you should be able to remove them all, but maybe I'm missing something.